### PR TITLE
Shieldtrait

### PR DESCRIPTION
**Pull request by MichaelCourtney**
https://www.freeorion.org/forum/viewtopic.php?f=6&t=11505

### CODE REVIEW

**Review by Oberlus — APPROVED** [review]
The +4 for multispectral seemed too much, but actually it's not OP and makes the part more interesting (for Trith specially).
The "+1 level" in the descriptions seems good enough.

**Comment by MichaelCourtney** [comment]
Switched to draft while till we get agreement in the thread on which species are good and bad

**Comment by MichaelCourtney** [comment]
Changed commit based on forum thread feedback

File: default/scripting/species/SP_ACIREMA.focs.txt [hunk 1]
```diff
@@ -5,7 +5,7 @@ Species
     Native
     CanProduceShips
 
-    tags = [ "SELF_SUSTAINING" "GREAT_INDUSTRY" "GOOD_RESEARCH" "GOOD_INFLUENCE" "GOOD_SHIELDS" "GREAT_SUPPLY" "PEDIA_SELF_SUSTAINING_SPECIES_CLASS" ]
+    tags = [ "SELF_SUSTAINING" "GREAT_INDUSTRY" "GOOD_RESEARCH" "GOOD_INFLUENCE" "ULTIMATE_SHIELDS" "GREAT_SUPPLY" "PEDIA_SELF_SUSTAINING_SPECIES_CLASS" ]
```
Comment (Grummel7) [inline default/scripting/species/SP_ACIREMA.focs.txt]: Acrimea also have a bonus to planetary shields. So far the tag GOOD_SHIELDS stood for both abilities.
I suggest align the tags with the effects, so using e.g. ULTIMATE_SHIP_SHIELDS here, add a tag GOOD_PLANETARY_SHIELDS.

Comment (MichaelCourtney) [reply default/scripting/species/SP_ACIREMA.focs.txt]: done

**Review by Grummel7 — APPROVED** [review]
Okay for me now.
I'll have to adopt the AI once this is merged, but that shouldn't take long.

**Pull request merged**